DOMjudge_tag_release.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+. release.env
+
+for i in `git tag | grep 7`; do git tag -d $i; done
+
+DOWNLOAD_RELEASE_SCRIPT=""
+
+notify_channel () {
+    # Optionally notify the Slack channel
+    # that a build has been triggered
+}
+
+test_tarball () {
+    # Happens as current side effect of docker build
+}
+
+build_deb () {
+    # Not sure if this should happen on the server?
+}
+
+build_docker () {
+    # Create on: https://gitlab.com/DOMjudge/domjudge-packaging/-/settings/ci_cd
+    curl -X POST \
+         -F token=$TOKEN \
+         -F "ref=master" \
+         -F "variables[DOMJUDGE_VERSION]=$1" \
+         https://gitlab.com/api/v4/projects/24097565/trigger/pipeline
+}
+
+process_tag () {
+    if [ ! -f $RELEASE_DIR/$1 ]; then
```

Comment:
I think you'd have to look for `$RELEASE_DIR/domjudge-$1.tar.gz`. It would also be helpful to assign `TAG=$1` for clarity.

Reply:
Agreed.